src/index.ts

```diff
@@ -44,3 +44,17 @@ export function toURL(deck: TypedDeck): string {
         passcodesToBase64(deck.extra) + "!" +
         passcodesToBase64(deck.side) + "!";
 }
+
+export function extractURL(query: string): string[] | null {
```

Comment:
Suggest renaming to `extractURLs` and `from` to better represent behaviour and remove `null` return.